src/configuration.ts

```diff
@@ -468,6 +468,11 @@ export interface INodeAttachConfiguration extends INodeBaseConfiguration {
   type: DebugType.Node;
   request: 'attach';
 
+  /**
+   * Inspector webSocket address
+   */
+  wsAddr?: string;
```

Comment:
Can you rename this to `websocketAddress` and add it to the `generate-contributions.ts` as well? The generation file is what is used to create the package.json contributions that contains the schema for the launch.json file.

Reply:
fixed.